migration/v0.32/README.md

```diff
@@ -36,6 +36,40 @@ As with all scripts in this repository `CK8S_CONFIG_PATH` is expected to be set.
 
 ## Prerequisites
 
+- If `.objectStorage.sync.enabled: true` and `.objectStorage.sync.syncDefaultBuckets: true` and `.thanos.objectStorage.type: swift` or `.harbor.persistence.type: swift` then the rclone jobs will automatically use swift for Thanos and/or Harbor source and destination buckets. You will need to create the application credentials for swift and add them into the `secrets.yaml`:
+
+    <details><summary>Create source application credentials</summary>
+
+    ```bash
+    source ${CK8S_CONFIG_PATH}/<source-openrc>.sh
+    source <(sops -d ${CK8S_CONFIG_PATH}/secret/<source-env-openstack-user>.sh)
+
+    openstack application credential create <env-name>-swift
+
+    sops ${CK8S_CONFIG_PATH}/secrets.yaml
+
+    objectStorage.swift.applicationCredentialID
+    objectStorage.swift.applicationCredentialSecret
+    ```
+
+    </details>
+
+    <details><summary>Create destination application credentials</summary>
+
+    ```bash
+    source ${CK8S_CONFIG_PATH}/<destination-openrc>.sh
+    source <(sops -d ${CK8S_CONFIG_PATH}/secret/<destination-env-openstack-user>.sh)
+
+    openstack application credential create <env-name>-swift
+
+    sops ${CK8S_CONFIG_PATH}/secrets.yaml
+
+    objectStorage.sync.swift.applicationCredentialID
+    objectStorage.sync.swift.applicationCredentialSecret
```

Comment:
Since these are not actually bash commands
```suggestion
    # set objectStorage.swift.applicationCredentialID in secres.yaml
    # set objectStorage.swift.applicationCredentialSecret in secres.yaml
    ```

    </details>

    <details><summary>Create destination application credentials</summary>

    ```bash
    source ${CK8S_CONFIG_PATH}/<destination-openrc>.sh
    source <(sops -d ${CK8S_CONFIG_PATH}/secret/<destination-env-openstack-user>.sh)

    openstack application credential create <env-name>-swift

    sops ${CK8S_CONFIG_PATH}/secrets.yaml

    # set objectStorage.sync.swift.applicationCredentialID in secres.yaml
    # set objectStorage.sync.swift.applicationCredentialSecret in secres.yaml
```

Reply:
spelling error secres.yaml should be secrets.yaml